include/drivers/gps.h

```diff
@@ -367,6 +367,28 @@ static inline int gps_deinit(struct device *dev)
 	return api->deinit(dev);
 }
 
+/**
+ * @brief Function to request A-GPS data.
+ *
+ * @param request Assistance data to request from A-GPS service.
+ * @param socket GPS socket to which assistance data will be written
+ *		 when it's received from the selected A-GPS service.
+ *		 If negative, GPS driver will be used to write the data instead
```

Comment:
Perhaps we should change this to `0` instead, as that's a value the GNSS socket can not have